script/compare.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	bolt "go.etcd.io/bbolt"
+	"log"
+)
+
+const (
+	vulnerabilityBucket = "vulnerability"
+)
+
+var (
+	oldBboltFile = flag.String("old_file", "cache/db/old.db", "old DB file")
+	newBboltFile = flag.String("new_file", "cache/db/new.db", "new DB file")
+)
+
+func main() {
+	flag.Parse()
+	oldVuln, oldAdvis := readFile(*oldBboltFile)
+	newVuln, newAdv := readFile(*newBboltFile)
+	fmt.Printf("got %v vulnerabilities and %v advisories from old DB\n", len(oldVuln), len(oldAdvis))
+	fmt.Printf("got %v vulnerabilities and %v advisories from new DB\n", len(newVuln), len(newAdv))
+	for k := range oldVuln {
+		if _, ok := newVuln[k]; !ok {
```

Comment:
Could you compare the value as well? You can use `reflect.DeepEqual` or other libraries such as https://github.com/google/go-cmp or https://github.com/kylelemons/godebug.